package.json

```diff
@@ -7,28 +7,28 @@
   "license": "MIT",
   "dependencies": {
     "bower": "~1.4.1",
-    "grunt": "~0.4.5"
-  },
-  "devDependencies": {
-    "connect-livereload": "~0.5.3",
     "connect-modrewrite": "~0.7.11",
-    "coveralls": "~2.11.2",
-    "grunt-bump": "~0.3.0",
+    "grunt": "~0.4.5",
     "grunt-contrib-clean": "~0.6.0",
```

Comment:
I would say all the libraries in `packages.json` are dev dependancies as they are all tools used for development rather then code that goes into our app.